src/pages/waiting-room-to-car/components/instructor-registration/instructor-registration.html

```diff
@@ -0,0 +1,15 @@
+            <ion-row class="mes-component-row mes-row-separator" id="instructor-registration-card" [formGroup]="formGroup">
```

Comment:
Indentation is off in this file. Probably worth turning on format on save in vscode, it would eliminate things like this (Settings -> Text Editor -> Formatting -> Format On Save).

Reply:
good suggestion, have turned auto formatting on.